end_to_end/test_convergence_1b_params.sh

```diff
@@ -26,7 +26,7 @@ done
 # Train
 export LIBTPU_INIT_ARGS="--xla_tpu_enable_data_parallel_all_reduce_opt=true --xla_tpu_data_parallel_opt_different_sized_ops=true --xla_tpu_enable_async_collective_fusion=true --xla_tpu_enable_async_collective_fusion_fuse_all_gather=true --xla_tpu_enable_async_collective_fusion_multiple_steps=true --xla_tpu_overlap_compute_collective_tc=true --xla_enable_async_all_gather=true"
 python3 MaxText/train.py MaxText/configs/base.yml run_name=$RUN_NAME\
-    steps=3400 per_device_batch_size=12.0 learning_rate=1e-3 enable_checkpointing=false\
+    steps=20400 per_device_batch_size=2.0 learning_rate=1e-4 enable_checkpointing=false\
```

Comment:
Is this change to the default training args needed and/or does it belong as part of this commit?

Reply:
This change is driven by other changes to the model. And unfortunately we aren't yet running this test on a continuous basis